middleware/encryptcookie/README.md

```diff
@@ -0,0 +1,5 @@
+# Encrypt Cookie Middleware
+
+Encrypt middleware for [Fiber](https://github.com/gofiber/fiber)
+
+TODO
```

Comment:
TODO for when I write these docs: cookie name is not encrypted

Reply:
@amir9480 please remove TODO